lib/puppet/provider/package/brewcask.rb

```diff
@@ -36,7 +36,7 @@ def self.legacy_caskroom
   end
 
   def self.new_caskroom
-    @new_caskroom ||= Pathname.new("#{home}/Caskroom")
```

Comment:
This needs to stay as double quotes otherwise it won't be evaluated and interpolated correctly.